queries.sql

```diff
@@ -164,3 +164,6 @@ WHERE v.name = 'Maisy Smith'
 GROUP BY s.name
 LIMIT 1;
 
+explain analyze select distinct(count(*)) from visits where animal_id = 4;
+explain analyze select distinct vet_id from visits where vet_id = 2
+explain analyze select email from owners where email = 'owner_19237@mail.com';
```

Comment:
- [ ] You should have included a pair of screenshots in the PR description with the results of the  EXPLAIN ANALYZE command for each of the above queries (in total it should be 6 screenshots):

The first screenshot should show how much time the query takes BEFORE the improvements you introduced.
- It the query planner output you should see the usage of Seq scan and/or Filter
- The execution time should be around 1 sec (1000 ms).
The second screenshot should show how much time the query takes AFTER the improvements introduced by student.
- The execution time should be significantly lower than 1 sec (1000 ms).

- Check if screenshots for each query prove that the execution time AFTER the improvements is lower than BEFORE the improvements.